pkg/app/web/src/modules/deployment-frequency.test.ts

```diff
@@ -0,0 +1,16 @@
+import { deploymentFrequencySlice } from "./deployment-frequency";
+
+describe("deploymentFrequencySlice reducer", () => {
+  it("should handle initial state", () => {
+    expect(
+      deploymentFrequencySlice.reducer(undefined, {
+        type: "TEST_ACTION",
+      })
+    ).toMatchInlineSnapshot(`
+      Object {
+        "data": Array [],
```

Comment:
nit: should it be `[]` 🤔

Reply:
This code is generated by `jest`, so we'll leave it as is.

Reply:
I see 😅 thanks